tuf/repository_tool.py

```diff
@@ -2551,10 +2566,17 @@ def add_target_to_bin(self, target_filepath, number_of_bins, custom=None):
       custom:
         An optional object providing additional information about the file.
 
+      fileinfo:
+        An optional fileinfo object, conforming to tuf.formats.FILEINFO_SCHEMA,
+        providing full information about the file.
+
```

Comment:
Given that we said that the only reason to have `custom` and `fileinfo`  is to support backwards compatibility (as the former can be passed within the latter) and `add_target_to_bin` didn't support this before, I think it is safe to not add `custom` here.

Reply:
I understand that my suggestion comes at the cost of consistency between `add_target` and `add_target_to_bin`. I am fine either way.

Reply:
I agree. As we plan to deprecate the `custom` parameter it doesn't make sense to _add_ it to `add_target_to_bin`. I've added a follow-on patch in 5ea1f63, though maybe I should have squashed it.